doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -175,6 +177,11 @@ is described in Chapter :ref:`syntaxextensionsandinterpretationscopes`.
    | ltac : ( @ltac_expr )
    field_def ::= @qualid {* @binder } := @term
 
+Note: many commands and tactics use :n:`@one_term` rather than :n:`@term`.
```

Comment:
What about using the `.. note::` directive?

Reply:
Eh, I'm not so keen on the look of the note directive--the colored box with contrasting header is very attention-getting, while most notes are distinctly secondary points, less important than the main text.

I think run-in titles in bold are a good option for notes and some other cases, like the example below.  They take less space and are easier for the reader to scan.

**Note**: many commands and tactics use one_term rather than term. The former need to be enclosed in parentheses unless they're very simple, such as a single identifier. This avoids confusing a space-separated list of terms with a term1 applied to a list of arguments.

Reply:
Well, in this particular case, it seems a pretty important note to me, so I would appreciate having the additional highlighting that this provides.

Reply:
I changed this for now but I may want to reconsider that in the future.